README.md

```diff
@@ -1,2 +1,102 @@
-# ES6_Awesome_Books
-Creation Book Library with ES6 Features
+# Awesome_Books
+Creation Book List Library for a user to store the list of books 
+
+> A demo website for Book Library.
+
+## Built With
+
+HTML and CSS
+Javascript : Arrays and Objects , DOM , Local Storage etc 
+
+## Video
+
+In the following [link](Coming Soon)
+
+
+## Live Server
+
+In the following link, you can view (Coming Soon)).
+
+## Screenshot
+
+![Book_Store_App(Page1)](./images/Image1.png)
+![Book_Store_App(Page2)](./images/Image2.png)
+![Book_Store_App(Page3)](./images/Image3.png)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+## Prerequisites
+
+### Browser
+
+Any modern browser works! :smile:
+
+
+### Knowledge
+
+Basic knowledge of the terminal and git
+
+
+### Getting Started
+
+To get a local copy follow these steps:
+
+1. Clone the repo by typing on your terminal
+
+```
+git clone git@github.com:RaviTejaM9602/Awesome_Books.git
+```
+
+2. Access the repo with
+
+```
+cd Conference
+```
+
+3. Clone the Repo or Download the Zip file
+Run the live server or simply open index.html with your browser.
+
+```
+code .
+```
```

Comment:
- [ ] You have done well 👍  by giving clear instructions on how to clone your project. However, I think you can also add instructions on how to setup your project right below this section, for example, a command `npm install` for installing npm packages. because freshers might also use your project for learning purposes in the future. And this will help them a lot.